pkg/controller/controller_test.go

```diff
@@ -1547,6 +1547,7 @@ func newTestController(t *testing.T, config fakeosb.FakeClientConfiguration) (
 		osb.LatestAPIVersion().HeaderValue(),
 		fakeRecorder,
 		7*24*time.Hour,
+		7*24*time.Hour,
```

Comment:
7 days here, or 20 minutes. Not that it really matters for the tests.

Reply:
it doesn't matter, maybe someone can just extract a sensible dummy value and reuse it while we're refactoring.